cs/remote/src/FASTER.common/NetworkSenderBase.cs

```diff
@@ -56,10 +58,7 @@ public NetworkSenderBase(int serverBufferSize)
         public abstract bool SendResponse(int offset, int size);
 
         /// <inheritdoc />
-        public abstract void SendResponse(byte[] buffer, int offset, int count, object context);
```

Comment:
i believe these abstracts are needed as they are used by other codes not in this repo.

Reply:
Reverted